docs/preservation_recreate.md

```diff
@@ -0,0 +1,39 @@
+# Preservation
+
+PDC Describe's preservation strategy for MVP release should allow for the following: 
+
+* A full application restore from the last system backup.
+* An individual object reassembly from backups using the DOI to locate files and metadata in cloud storage.
+
+## Files and directories
+
+* Object files are stored in Amazon S3 under a directory structure based on their DOIs.  
+* Objects can be retrieved from S3 using the DOI to determine location. 
+* DataCite XML, object metadata from Rails as JSON, and provenance metadata as JSON are exported to a sidecar directory called `princeton_data_commons` within the DOI directory for the object in S3.
+
+### Example Backup Directory Structure:
+
+For example, assuming that this is the DOI for the object that needs to be recreated, `10.80021/1tzs-ph44`:
+
+```
+- s3:/bucket/10.80021/1tzs-ph44/42/
+  * file1.png 
+  * dataset.zip 
+  * princeton_data_commons/
+    * datacite.xml
+    * metadata.json 
+    * provenance.json
+```
+
+## Recreating an object from S3
+
+1. Locate the DOI folders in the appropriate bucket is S3.
+1. Download all of the files within the folder associated with the object, including the contents of the `princeton_data_commons` folder, which should contain three files as noted in the example directory structure aboveabove.
+1. Repopulate the metadata field values in the PDC Describe interface using the contents of `metadata.json`.  This file contains non-markdown formatting information such as line breaks that do not belong in the `datacite.xml` file.
```

Comment:
We should add a note here to indicate that if the `metadata.json` was generated with a version of the Work that is not compatible with the current version in the system (e.g. `metadata.json` was produced 10 months ago and PDC Describe has new fields and perhaps some of them mandatory) then the restore process won't be an easy process. Doable for sure, but not easy.